### PR TITLE
Hysterical-Horses

### PR DESCRIPTION
**Pull request by fisher60**

### CODE REVIEW

**Comment by SebastiaanZ** [comment]
No merge issues found.

No flake8 linting errors found.

File: hysterical-horses/code_jam/blogs/views.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        try:
+            parent_id = int(self.request.POST.get("parent_id"))
+        except Exception as e:
+            print(e)
+            parent_id = None
+
+        return parent_id
+
+    def post(self, request, *args, **kwargs):
+        print("POST METHOD RUNNING")
```
Comment (Den4200) [inline hysterical-horses/code_jam/blogs/views.py:73]: Debug prints should be removed.

File: hysterical-horses/code_jam/blogs/views.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+            new_comment = comment_form.save(commit=False)
+            new_comment.post = comment_post
+            new_comment.author = request.user
+            new_comment.save()  # Save to database
+            return HttpResponseRedirect(self.request.path_info)  # Refresh
+
+    def get_context_data(self, **kwargs):
+        """Runs when GET is called."""
+        print("CONTEXT DATA RUNNING")
```
Comment (Den4200) [inline hysterical-horses/code_jam/blogs/views.py:97]: This debug print should be removed as well.

File: hysterical-horses/code_jam/users/apps.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class UsersConfig(AppConfig):
+    name = "users"
+
+    def ready(self):
+        import users.signals
+        print(users.signals.__name__)
```
Comment (Den4200) [inline hysterical-horses/code_jam/users/apps.py:9]: This print should be removed as well, but I see why you did this - to pass linting tests. However, you should have just added a `# NOQA: F401` to ignore the unused import error.
```suggestion
        import users.signals  # NOQA: F401
```

**Pull request merged**